netjsonconfig/backends/openwrt/converters/interfaces.py

```diff
@@ -193,6 +225,17 @@ def _intermediate_vxlan(self, interface):
         interface['vid'] = interface.pop('vni')
         return interface
 
+    def _intermediate_8021_vlan(self, interface):
+        interface['name'] = '{}.{}'.format(interface['ifname'], interface['vid'])
+        interface['.name'] = '{}_{}'.format(interface['.name'], interface['vid'])
```

Comment:
First of all I want to thank you for these features! Really appreciated.
I just have a stupid question: why formatting the name in this way?
For example if I use as logical interface name "CORPORATE_WAN" with vid 201 I'll get VLAN_CORPORATE_WAN_201. 
I think that is better to let the user choose the full name.

```suggestion
```